api/tests/Feature/AuthenticationTest.php

```diff
@@ -2,14 +2,7 @@
 
 declare(strict_types=1);
 
-use function Eerison\PestPluginApiPlatform\{
```

Comment:
Hi @eleandrolima 
Is there any reason to remove this? I guess it's doing the tests fail